pkg/gce-pd-csi-driver/node_test.go

```diff
@@ -86,3 +88,270 @@ func TestNodeGetVolumeLimits(t *testing.T) {
 		}
 	}
 }
+
+func TestNodePublishVolume(t *testing.T) {
+	gceDriver := getTestGCEDriver(t)
+	ns := gceDriver.ns
+	testCases := []struct {
+		name       string
+		req        *csi.NodePublishVolumeRequest
+		expErrCode codes.Code
+	}{
+		{
+			name: "Valid request",
+			req: &csi.NodePublishVolumeRequest{
+				VolumeId:          "1",
```

Comment:
can we use a valid volume ID here of the form `projects/{project}/zones/{zone}/disks/{name}`
technically I think we want to validate the ID format and return an invalidargument error if it is not in that format (although right now we don't do that)

Reply:
ditto for the other tests as well

Reply:
OK. I created several const variables for default testing volume ID, staging path and target path.